packages/flutter/test/rendering/editable_gesture_test.dart

```diff
@@ -11,18 +11,17 @@ import 'package:leak_tracker_flutter_testing/leak_tracker_flutter_testing.dart';
 void main() {
   final TestWidgetsFlutterBinding binding = _GestureBindingSpy();
 
-  testWidgets('attach and detach correctly handle gesture',
-  // TODO(polina-c): clean up leaks, https://github.com/flutter/flutter/issues/134787
-  experimentalLeakTesting: LeakTesting.settings.withIgnoredAll(),
-  (_) async {
+  testWidgets('attach and detach correctly handle gesture', (_) async {
     expect(WidgetsBinding.instance, binding);
     final TextSelectionDelegate delegate = FakeEditableTextState();
+    final ViewportOffset offset = ViewportOffset.zero();
+    addTearDown(offset.dispose);
     final RenderEditable editable = RenderEditable(
       backgroundCursorColor: Colors.grey,
       selectionColor: Colors.black,
       textDirection: TextDirection.ltr,
       cursorColor: Colors.red,
-      offset: ViewportOffset.zero(),
+      offset: offset,
```

Comment:
I'm not too sure about this one, should `editable.dispose` dispose its `offset` ?

Reply:
Nope, editable did not create offset, so it should not dispose it. So, your way to do it is correct.